ReleaseProcessScript/packages.config

```diff
@@ -1,4 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="GitForWindows" version="2.26.2" targetFramework="net461" />
```

Comment:
What's up with this dependency? Up until now, we did not need a pre-defined git-dependency but used the one from the client project's context.

Reply:
Okay, from what I can tell, the tests are part of the production code. the NuGet dependency remain a problem specified this way because it will be included in the finished package. there's a developmentonly-flag available that should take care of business.